src/bootstrap/src/core/builder/cargo.rs

```diff
@@ -1133,7 +1133,7 @@ impl Builder<'_> {
             cargo.env("RUSTC_BACKTRACE_ON_ICE", "1");
         }
 
-        if self.is_verbose() {
+        if self.is_verbose_than(1) {
```

Comment:
Huh, that's a weird (pre-existing) function name. I had to look at the code to understand what it meant. `is_more_verbose_than` would be clearer. Even clearer would be just removing it and using `if self.verbosity > 1` directly. The related `verbose_than` function could also be removed.

Reply:
Yeah, the names are pretty terrible here, I plan to rename them, once I get to it.